### DISCUSSION TITLE
can you do something like (xbox controller) Y + B = V (on keyboard)

### DISCUSSION THREAD

**Discussion by redwolf460** [AntiMicroX/antimicrox#786] [Q&A]
I have been struggling I want to set my games up like Xeno verse 2

**Comment by mrbeauregarde** [comment]
Im sure theres a more sophisticated answer, but it would be easy enough to have the (Y) button hold a set [S], where [S] defines the binding on (B) as (keyboard V). Depending on how many pairs you need, this could be untenable, but the concept should be mutable to your use case.
